WordPress/Classes/ViewRelated/System/Notices/NoticePresenter.swift

```diff
@@ -23,8 +38,8 @@ class NoticePresenter: NSObject {
         }
     }
 
-    @objc convenience init(presentingViewController: UIViewController) {
-        self.init(store: StoreContainer.shared.notice, presentingViewController: presentingViewController)
+    @objc override convenience init() {
```

Comment:
With the change in the signature of this initializer, I think you can remove `@objc` from this signature..